app/controllers/lesson_views_controller.rb

```diff
@@ -0,0 +1,19 @@
+class LessonViewsController < ApplicationController
+  def add_view
+    @lesson_view = LessonView.new(ip: client_ip, lesson_id: params[:lesson_id], user_id: current_user_id)
+    if @lesson_view.save
+      render :show
+    else
+      render :error, status: :unprocessable_entity
+    end
+  end
+
+  private
+
+  def current_user_id
+    @decoded = JsonWebToken.decode(jwt_token)
+    @current_user_id = User.find_by(email: @decoded['email']).id
+  rescue StandardError
```

Comment:
Why we are tracking only StandardError here? You can use just rescue without any type of error. It will handle all types of the errors.

Reply:
It's rubocop autocorrect. First I had used just rescue. I'll return just rescue, but I don't understand why rubocop fixed it. 
![image](https://user-images.githubusercontent.com/92786798/205902078-f02ec477-d4cb-4b04-bde8-a7d1358b75ab.png)

Reply:
Maybe StandardError is all the errors?